service-account/bound-sa-token.md

```diff
@@ -1,122 +0,0 @@
-%title: Bound Service Account Token
-%author: Krzysztof Ostrowski
-%date: 10.03.2023
-
--> # Bound Service Account Token <-
-
---------------------------------------------------
--> # Table of Contents <-
-==============
-
--> # What is a Service Account Token  <-
--> # Service Account Token as a YAML <-
--> # Old Service Account Token Behavior  <-
--> # What is a Bound Service Account Token  <-
--> # Lord of Tokens: Kubelet <-
--> # Lord of ServiceAccounts: Service Account Admission Controller <-
--> # Bound Service Account Token in use by a Pod <-
-
--------------------------------------------------
--> # What is a Service Account Token  <-
-
-An **identity** that an instance or an application can use to make **API requests**:
-
-1. Intended to be used for **processes**.
-2. Bound to a **namespace**.
-3. Being associated with a ServiceAccount provides a **Bearer Token**.
-
--------------------------------------------------
--> # Service Account Token as a YAML <-
-
-```yaml
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: my-sa
-  namespace: my-namespace
-```
-
--------------------------------------------------
--> # Old Service Account Token Behavior  <-
-
-
-1. Previously **didn't expire**.
-2. Previously could be **used anywhere**.
-3. Token was **stored in a secret**.
-
--------------------------------------------------
--> # What is a Bound Service Account Token  <-
-
-- A Service Account Token that is **bound** to an **audience and lifespan**.
```

Comment:
That's not what "bound" means. The "bound" in the name is meant to outline the binding of the token to a specific resource.

Reply:
I guess we are both correct, even though I would still consider "audiences" and "token expiration" just generic attributes of a token, rather than calling them a binding.

Source: https://github.com/kubernetes/enhancements/blob/c0738c69da3048e602efee6f75819c4ca9e6f26f/keps/sig-auth/1205-bound-service-account-tokens/README.md#tokenrequest